### PR TITLE
#52 fix translation

### PR DESCRIPTION
**Pull request by tuanlc**

### CODE REVIEW

**Comment by piavgh** [comment]
There are conflicting files:

> src/linagora.esn.unifiedinbox/i18n/en.json
src/linagora.esn.unifiedinbox/i18n/fr.json
src/linagora.esn.unifiedinbox/i18n/ru.json
src/linagora.esn.unifiedinbox/i18n/vi.json
src/linagora.esn.unifiedinbox/i18n/zh.json

**Comment by huy-ta** [comment]
Where do these `{{ text }}` come from? I don't see them reflected in the changed code.

File: src/linagora.esn.unifiedinbox/i18n/fr.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "\"%s\"": "«&nbsp;%s&nbsp;»",
-  "%s items": "%s éléments",
+  "\"%s\"": "\"{{ value }}\"",
```
Comment (piavgh) [inline src/linagora.esn.unifiedinbox/i18n/fr.json:2]: In the old code, there is a « and a » :rofl:

**Comment by tuanlc** [comment]
> Where do these {{ text }} come from? I don't see them reflected in the changed code.
 
Here: src/linagora.esn.unifiedinbox/app/services/mailboxes-filter/mailboxes-filter-service.js

File: src/linagora.esn.unifiedinbox/i18n/fr.json (hunk 1 above)
Comment (tuanlc) [reply src/linagora.esn.unifiedinbox/i18n/fr.json:2]: I think it was wrong since other languages respect the original version to use `"` and `"`

**Pull request merged**